angular-notify.js

```diff
@@ -57,19 +66,26 @@ angular.module('cgNotify', []).factory('notify',['$timeout','$http','$compile','
                 angular.element(args.container).append(templateElement);
                 messageElements.push(templateElement);
 
-                if (args.position === 'center'){
-                    $timeout(function(){
-                        templateElement.css('margin-left','-' + (templateElement[0].offsetWidth /2) + 'px');
-                    });
+                switch (args.position){
+                    case 'center':  templateElement.addClass('cg-notify-message-center');
+                                    templateElement.css('margin-left','-' + (templateElement[0].offsetWidth /2) + 'px');
```

Comment:
I've found that wrapping this line in a $timeout call (as was being done previously) is actually important. Without that, the "center" case of positioning stopped working for me:

Before:
---------

![working](https://cloud.githubusercontent.com/assets/868639/5898235/8487ebf2-a534-11e4-9685-5cfa303646d0.png)

After:
---------
![not-working](https://cloud.githubusercontent.com/assets/868639/5898236/84891b08-a534-11e4-9e1a-6d10dbb1bbae.png)